### DISCUSSION TITLE
Block Posts Functionality

### DISCUSSION THREAD

**Discussion by HydraheadHunter** [AprilSylph/XKit-Rewritten#820] [Suggestion Box]
Currently blocked posts appear in a list with an ID# and the option to unblock them. I'd like a bit more functionality when blocking posts. Mostly I want X things

1. A "See It" link to the post to see what I blocked.
2. The ability to label blocked posts as they appear in the list, like labeling post 4656648654 or whatever as "Hat Possession."
3. The ability to say yes, I'm really sure I never want to see this post again, don't even include it in the list of blocked posts.

**Comment by JRHB314** [comment]
Agree with #1, for sure. 

I don't care as much about 2 or 3, but I do think it would be nice to have something other than post id to identify the post. Maybe the "[username]: [beginning of post]" ("bogleech: Twitter decaying before our eyes not...") description that tumblr sometimes uses for posts? Or else something at least vaguely descriptive, like "[OP username]'s [type] post" e.g. "user123's text post".
